src/content/security/index.md

```diff
@@ -180,7 +180,7 @@ These scams usually stipulate a limited time of opportunity to claim the giveawa
 
 #### Social media hacks {#social-media-hacks}
 
-A high-profile version of this occurred in July 2020, when the Twitter accounts of prominent celebrities and organizations got hacked. The hacker simultaneously posted a Bitcoin giveaway on the hacked accounts. Although the deceptive tweets were quickly noticed and deleted, the hackers still managed to get away with 11 bitcoin (or $500,000 as of September 2021)
+A high-profile version of this occurred in July 2020, when the Twitter accounts of prominent celebrities and organizations got hacked. The hacker simultaneously posted a Bitcoin giveaway on the hacked accounts. Although the deceptive tweets were quickly noticed and deleted, the hackers still managed to get away with 11 bitcoins (or $500,000 as of September 2021)
```

Comment:
@wackerow any thoughts here?

Reply:
Traditionally have stuck with "bitcoin" for both singular and plural. 

https://github.com/bitcoinbook/bitcoinbook

Going off this respected resource as well, no use of "bitcoins" in the entire book.

Would also go with lowercase "bitcoin" for the asset (ie: above should be `...posted a bitcoin giveaway...`), and only capitalize it when referring to the protocol or network. ie. "I sent 2 bitcoin on the Bitcoin network."

Reply:
Thanks for the info 😊 
Changing it to bitcoin then.